app/services/rapid_ready_for_decision/hypertension_special_issue_manager.rb

```diff
@@ -9,8 +9,8 @@ def initialize(submission)
     end
 
     def add_special_issue
-      submission_data = JSON.parse(submission.form_json)
-      disabilities = submission_data.dig('form526', 'form526', 'disabilities')
+      submission_data = JSON.parse(submission.form_json, symbolize_names: true)
```

Comment:
Reverts [this change](https://github.com/department-of-veterans-affairs/vets-api/pull/9181/files#r812406293)